RubberduckTests/Inspections/ConstantNotUsedInspectionTests.cs

```diff
@@ -11,130 +11,6 @@
 namespace RubberduckTests.Inspections
 {
     [TestClass]
-    public class ObjectVariableNotSetInspectionTests
```

Comment:
That's entirely my bad. Sorry about this, I'll be more careful about extracting classes to their own file going forward.